# GP Parameters
float32[] alpha
float32[] params
float32[] xData
float32[] yData
float32[] zData
float32[] yawData
